### ISSUE TITLE
Request to add the undo shortcut.

### ISSUE THREAD

**Issue by sd309991** [brian-girko/bookmarks-commander#22]
Hi Brian

Is it possible to undo a delete? I will sometimes select the wrong bookmark and delete it. When I am aware something is missing, it is too late. Or it is possible to find the delete link somewhere on the PC?

Yours

Sam

**Comment by sd309991** [comment]
Thank you!

**Comment by brian-girko** [comment]
This is a must-have feature. However, there is no built-in function for it. I am still exploring options.

**Comment by kevinfrteels** [comment]
Yes. Two years later. Haven't had time to look elsewhere and see if it's been added. i installed the add-in and there was a link that simply said "Feature request to add undo shortcut" which landed me on this page when I clicked. Gonna poke around now. lol
